shortfin/python/shortfin_apps/llm/components/generate.py

```diff
@@ -203,7 +204,7 @@ async def run(self):
                 error_message="Server queue is full. Please try again later.",
                 code=ResponderErrorCodes.QUEUE_FULL,
                 extra_fields={
-                    "current_size": self.service.queue_manager.current_queue_size,
+                    "current_size": self.service.queue_manager._current_queue_size,
```

Comment:
We generally don't want to access privately owned member variables (e..g. prefixed with a `_`).